### PR TITLE
qemu 5.2.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

**Comment by fxcoudert** [comment]
`ERROR: Cannot find Ninja`

**Comment by chenrui333** [comment]
yeah, I tried adding ninja in my local, and still did not build :(

**Comment by zeha** [comment]
you need:

```patch
+  depends_on "ninja" => :build
+  depends_on "meson" => :build
```

and

- https://github.com/zeha/qemu/commit/7f2fa31e11dd10b4436d1546a8afad52a41ab581
- (optionally) https://github.com/zeha/qemu/commit/5a1b1c113e2a630dd2a4e9c785326365979cafe6

these are not stable URLs

**Comment by tiiiecherle** [comment]
Thanks for updating this.
Is virtfs enabled for this build? If not it would be nice if it could be enabled as it helps a lot sharing folders to linux guests.
I was trying UTM.app and virtfs is disabled there.
Thanks

File: Formula/qemu.rb [hunk 1]
```diff
@@ -62,7 +63,8 @@ def install
     end
 
     system "./configure", *args
-    system "make", "V=1", "install"
+    system "ninja", "-v"
```
Comment (roolebo) [inline Formula/qemu.rb:66]: This hunk is invalid. There's a dependency on ninja but it shouldn't be used directly yet.

**Review by roolebo — CHANGES_REQUESTED** [review]

**Comment by r58Playz** [comment]
[This](https://gist.github.com/niw/e4313b9c14e968764a52375da41b4278) may be helpful(there's an english version below)

**Comment by tiiiecherle** [comment]
Does anyone have a working local build to share or are there any news how to fix this? I tried but couldn't get it to work...
Thanks for any help.

**Comment by ziyi-yan** [comment]
> Does anyone have a working local build to share or are there any news how to fix this? I tried but couldn't get it to work...
> Thanks for any help.

I followed the advice from @zeha and made two patches to fix it. At least it build successfully on my machine now.

**Comment by zeha** [comment]
Upstream URLs are now:

https://git.qemu.org/?p=qemu.git;a=patch;h=0dbce6efb5ff2e7113734d3a0cabbf87fc56feec
https://git.qemu.org/?p=qemu.git;a=patch;h=cb7abd8319d19000b57ae6c5c474c2635db054c6

Haven't tried applying them on top of 5.2.0 directly.

**Review by carlocab — APPROVED** [review]
🎉

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**